currency_exchanger/currency_exchanger/request.py

```diff
@@ -0,0 +1,12 @@
+import requests
+import secrets
+
+payload = {}
+headers= {
+  "apikey": secrets.API_TOKEN
+}
+
+response = requests.request("GET", secrets.BASE_CURRENCY_URL, headers=headers, data = payload)
```

Comment:
As a said before, make entire thing a customizable function. Functions start with "def" in python. Make a function that takes in (link, token) arguments, and returns a json. This will allow us to call this function from other files, and pass secrets.BASE_URL and secrets.API_TOKEN as arguments.